gb-cpu/src/microcode/dec.rs

```diff
@@ -5,7 +5,7 @@ use super::{
 use crate::interfaces::WriteFlagReg;
 
 pub fn dec16(ctl: &mut MicrocodeController, _state: &mut State) -> MicrocodeFlow {
-    let value = ctl.pop_u16() + 1;
+    let (value, _) = ctl.pop_u16().overflowing_sub(1);
```

Comment:
maybe logging overflows would be a good idea?

Reply:
I don't know if this would be useful